src/components/ErrorPages/DefaultErrorPage.tsx

```diff
@@ -56,10 +59,11 @@ export default function ErrorPage({
           <br />
           <br />
           <Button
-            onClick={() =>
+            onClick={() => {
               // Refresh the Page
-              window.location.reload()
-            }
+              goBack();
```

Comment:
Let's add a fallback url ("/") in case user encounters error on opening link in a new tab (in which case, there would be no history).